identity/app/model/EmailSubscriptions.scala

```diff
@@ -3,13 +3,15 @@ package model
 case class EmailSubscriptions(subscriptions: List[EmailSubscription])
 case class EmailSubscription(
   name: String,
+  subheading: Option[String] = None,
```

Comment:
Would it be possible to put this at the end of the parameter list with the others or remove the default and fill it out in each case?
I would try and stick to defaults at the end of the parameter list of case classes as I've seen odd behaviour with inference/macros when done like this.